src/web/src/app/space/detail/component.html

```diff
@@ -0,0 +1,113 @@
+
+<div *ngIf="currentSpaceLoaded">
+	<h1>{{currentSpace.name}}</h1>
+</div>
```

Comment:
А зачем отдельный div ? И зачем currentSpaceLoaded, можно же проверить сам объект?
```suggestion
<h1 *ngIf="currentSpace">{{currentSpace.name}}</h1>
```

Reply:
Готово.